client/src/components/LoadClients/index.js

```diff
@@ -43,11 +60,11 @@ const LoadClients = props => {
           <div>
             <ul>
               {
-                loadClients.map((lc, index) => (
+                sortData.map((lc) => (
                     <li key={lc.id}>
                       <button className={`loadclient ${excuteClients[lc.id] ? "selected" : ""}`} onClick={() => handleToggleSelected(lc.id)}>{lc.name}</button>
                       <div className="divider"></div> 
-                      <button className={`load-client-status ${lc.currstatus}`} title={lc.currstatus} onClick={() => props.handleOpen(index)}></button>
+                      <button className={`load-client-status ${lc.currstatus}`} title={lc.currstatus} onClick={() => props.handleOpen(lc.id)}></button>
```

Comment:
I don't think load client details is using ID

Reply:
Could you clarify what you are referring to here? The only thing I am seeing related to load client details is the handleOpen function which uses ID